website/docs/getting-started/cluster-setup/openshift.md

```diff
@@ -0,0 +1,58 @@
+---
+title: OpenShift
+sidebar_position: 2
+---
+
+# Setup an OpenShift cluster
+*Note that this guide is only helpful in setting up a development environment; this setup is not recommended for a production environment.*
+
+## Prerequisites
+* This guide assumes that you have an OpenShift cluster setup, this could also be a [crc](https://crc.dev/crc/#installing-codeready-containers_gsg) cluster.
```

Comment:
```suggestion
* This guide assumes that you have an OpenShift cluster setup, this could for example be a [crc](https://crc.dev/crc/#installing-codeready-containers_gsg) cluster.
```

(crc being an OpenShift cluster)